Dockerfile

```diff
@@ -1,21 +1,26 @@
-FROM debian:jessie
+FROM debian:stretch
 
 MAINTAINER buildmaster@rocket.chat
 
+RUN apt-get update \
+	&& apt-get install -y \
+		gpg \
+    && rm -rf /var/lib/apt/lists/*
+
 # gpg: key 4FD08014: public key "Rocket.Chat Buildmaster <buildmaster@rocket.chat>" imported
 RUN gpg --keyserver ha.pool.sks-keyservers.net --recv-keys 0E163286C20D07B9787EBE9FD7F9D0414FD08104
 
 # gpg keys listed at https://github.com/nodejs/node
 RUN set -ex \
  && for key in \
-      9554F04D7259F04124DE6B476D5A82AC7E37093B \
       94AE36675C464D64BAFA68DD7434390BDBE9B9C5 \
-      0034A06D9D9B0064CE8ADF6BF1747F4AD2306D93 \
       FD3A5288F042B6850C66B31F09FE44734EB7990E \
       71DCFD284A79C3B38668286BC97EC7A07EDE3FC1 \
       DD8F2338BAE7501E3DD5AC78C273792F7D83545D \
-      B9AE9905FFD7803F25714661B63B535A4C206CA9 \
       C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8 \
+      B9AE9905FFD7803F25714661B63B535A4C206CA9 \
+      56730D5401028683275BD23C23EFEFE93C4CFFFE \
+      77984A986EBC2AA786BC0F66B01FBB92821C587A \
```

Comment:
Why change the gpg key?  This shouldn't have changed

Reply:
Keys at https://github.com/nodejs/node have changed.

Reply:
ok also updated to have current list.  Seems to have changed even more.